src/BaseBuilder.js

```diff
@@ -38,7 +39,7 @@ export default class BaseBuilder {
    * @param {boolean} disabled
    * @return {Builder}
    */
-  setDisabled(disabled) {
+  setDisabled(disabled = true) {
```

Comment:
Can you update the docs to include this default?

Reply:
Done